action.yml

```diff
@@ -14,37 +14,53 @@ inputs:
     description: Secret Scanning Test Suite Mode
     default: validate
 
+  templates-path:
+    description: Path to template directory
+
   template-main:
-    description: Main README Template
+    description: Main README template name
   
   template-patterns:
-    description: Main Pattern Template
+    description: PATTERNS template name
 
   argvs:
-    description: 'Additional Arguments'
+    description: Additional arguments
 
 
 runs:
   using: "composite"
   steps:
-    - uses: actions/setup-python@v5
+    - name: Setup Python
```

Comment:
We had a few issues with setting up Python this way on Enterprise Server and using self-hosted runners. I recommend we remove this line and add to the docs that Python 3.9+ needs to be installed on the runner

https://github.com/advanced-security/policy-as-code/issues/69